tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -107,6 +107,7 @@ sub post_fail_hook {
 
     #Restore Network setting
     virt_autotest::virtual_network_utils::restore_network($virt_host_bridge, $based_guest_dir);
+    $self->SUPER::post_fail_hook;
```

Comment:
Thanks for the effort! 

With calling SUPER::post_fail_hook as last step, which is after restart_libvirtd, destroy_virt_network, restore_guests, etc, many important instant state of host and guest will be lost when collecting host and guest supportconfig. So I am afraid it needs to be called immediately after test fail as first step in post_fail_hook, IF it does not affect later lines from L93~L109. One important impact of it will be to virt_autotest::virtual_network_utils::upload_debug_log(). 
- Please @guoxuguang help evaluate if the uploaded logs of this step is still valid.
- Besides, what logs are duplicated between virt_autotest::virtual_network_utils::upload_debug_log() and SUPER::post_fail_hook. Anything can be optimized here to avoid redundancy?